corehq/apps/toggle_ui/utils.py

```diff
@@ -1,7 +1,108 @@
+from couchexport.writers import Excel2007ExportWriter
+from django.utils.translation import ugettext as _
+from io import BytesIO
 from corehq.toggles import all_toggles
 
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.es import UserES
+from corehq.util.quickcache import quickcache
+
 
 def find_static_toggle(slug):
     for toggle in all_toggles():
         if toggle.slug == slug:
             return toggle
+
+
+@quickcache(['domain'], timeout=10)
+def get_subscription_info(domain):
+    subscription = Subscription.get_active_subscription_by_domain(domain)
+    if subscription:
+        return subscription.service_type, subscription.plan_version.plan.name
+    else:
```

Comment:
@snopoke Mmm...I take it you mean that the whole `else` clause could simply be replaced with `return None, None` instead of returning `None, None` _whithin_ an else clause? If so, then yes, I can do that. If not (else, hehe) and you mean only remove the else clause completely, then the code would...break...when unpacking `None`...

Reply:
yea, I meant just `return None, None` with not `else`